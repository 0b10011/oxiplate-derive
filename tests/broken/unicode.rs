use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "\u{1}\u{12}\u{123}\u{5432}{{ foo }}\u{1}\u{12}\u{123}\u{5432}"]
struct Unicode {}

fn main() {
    let data = Unicode {};

    print!("{}", data);
}
