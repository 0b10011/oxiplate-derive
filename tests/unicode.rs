use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate = "unicode.html.oxip"]
struct Data {
    foo: &'static str,
}

#[test]
fn external_unicode() {
    let template = Data { foo: "bar" };

    assert_eq!(format!("{}", template), "bar‚ùØ\n");
}
