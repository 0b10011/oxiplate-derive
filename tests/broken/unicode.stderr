error[E0609]: no field `foo` on type `&Unicode`
 --> tests/broken/unicode.rs:4:51
  |
4 | #[oxiplate_inline = "\u{1}\u{12}\u{123}\u{5432}{{ foo }}\u{1}\u{12}\u{123}\u{5432}"]
  |                                                   ^^^ unknown field
